### ISSUE TITLE
Two class object detection

### ISSUE THREAD

**Issue by pearlmary** [apple/ml-cvnets#36]
Hi Team,
For a two class object detection problem using mobileVITv2, I'm getting very low mAP and worse classification accuracy. 
Can you suggest me any hyperparameters I should change? 
Exactly, what changes I should make in this code. I have used the same coco detection example but I've given only two classes for detection instead of 81.

**Comment by sacmehta** [comment]
Hi @pearlmary ,  MobileViTv2 does not do anything special for object detection. It uses SSD head for the detection task. So, it should work on any dataset. It would be good if you can also try few other detection heads and see if you are observing similar issues. 

With the limited information that is provided, it appears that dataset is imbalanced. You probably want to look into dataset statistics and see if it is the case. If so, you probably want to modify the loss functions or use under-/over-sampling methods to address this imbalance.

Hope this helps.
